hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -458,7 +458,7 @@ public final void readData(ObjectDataInput in) throws IOException {
     @Override
     public String toString() {
         final StringBuilder sb = new StringBuilder(getClass().getName()).append('{');
-        sb.append("serviceName='").append(serviceName).append('\'');
+        sb.append("serviceName='").append(getServiceName()).append('\'');
```

Comment:
What is the added value of calling the getter? It is a field in the Operation without any form of logic. Also it isn't in line with the other fields being used in this toString method.